### PR TITLE
Feature/105 update person page

### PR DESCRIPTION
**Pull request by todd-dembrey**
Funding is something that we may start to pull in automatically based on assigning the user account to the person profile.

### CODE REVIEW

File: opentech/public/people/templates/people/person_page.html [hunk 1]
```diff
@@ -52,6 +52,18 @@ <h2>Social Media Links</h2>
...
+            <table>
+                <tr>
+                    <td>{{ funding.year }}</td>
+                    <td>${{ funding.value }}</td>
+                    <td>{{ funding.duration }} months</td>
+                    <td><a href="{% pageurl funding.source %}">{{ funding.source }}</a></td>
+                </tr>
+            </table>
+            {% endfor %}
+            <p>Total FUnding: {{ page.total_funding }}</p>
```
Comment (zerolab) [inline opentech/public/people/templates/people/person_page.html]: `s/FUnding/Funding/`

File: opentech/public/people/templates/people/person_page.html [hunk 2]
```diff
@@ -23,11 +23,11 @@ <h1>{{ page.first_name }} {{ page.last_name }}</h1>
                 <p>{{ page.email }}</p>
             {% endif %}
 
-            {% with phone_numbers=page.phone_numbers.all %}
-                {% if phone_numbers %}
+            {% with contact_details=page.contact_details.all %}
```
Comment (zerolab) [inline opentech/public/people/templates/people/person_page.html]: where does email feature here?

Comment (todd-dembrey) [reply opentech/public/people/templates/people/person_page.html]: Two lines up 

https://github.com/OpenTechFund/opentech.fund/pull/29/files#diff-2b4243313d52162d0c1c28c4c0a3f345R23

Comment (zerolab) [reply opentech/public/people/templates/people/person_page.html]: /me facepalm

**Comment by zerolab** [comment]
LGTM

**Pull request merged**